updater/updater_test.go

```diff
@@ -0,0 +1,279 @@
+package updater
+
+import (
+	"bytes"
+	"context"
+	"encoding/xml"
+	"errors"
+	"log"
+	"os"
+	"testing"
+
+	omahaSpec "github.com/kinvolk/go-omaha/omaha"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"gopkg.in/guregu/null.v4"
+
+	"github.com/kinvolk/nebraska/backend/pkg/api"
+	"github.com/kinvolk/nebraska/backend/pkg/omaha"
```

Comment:
I guess that should go together as it's a separate Go module?
```suggestion
	"github.com/stretchr/testify/require"
	"gopkg.in/guregu/null.v4"
	"github.com/kinvolk/nebraska/backend/pkg/api"
	"github.com/kinvolk/nebraska/backend/pkg/omaha"
```